src/wp-admin/includes/network.php

```diff
@@ -698,7 +698,17 @@ function network_step2( $errors = false ) {
 		);
 		echo '</p></li>';
 
-	else : // End $is_nginx. Construct an .htaccess file instead:
+	elseif ( $is_nginx_unit ) : // End $is_nginx, check for NGINX Unit:
+
+		echo '<li><p>';
+		printf(
+			/* translators: %s: Documentation URL. */
+			__( 'It seems your network is running within Nginx UNIT. <a href="%s">Learn more about further configuration</a>.' ),
```

Comment:
Isn't this capitalization backwards?

```suggestion
			__( 'It seems your network is running within NGINX Unit. <a href="%s">Learn more about further configuration</a>.' ),
```

(https://unit.nginx.org/)